### PR TITLE
Removed upload/auth functionalities

### PR DESCRIPTION
**Pull request by salmanm**
Removed following features

- Upload
- Login
- Logout
- Check user

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -96,5 +87,5 @@ and that a record of the contribution (including all personal information I
 submit with it, including my sign-off) is maintained indefinitely and may be
 redistributed consistent with this project or license(s) involved.
 
-[CoC]: CODE_OF_CONDUCT.md
-[DCO]: https://developercertificate.org/
+[coc]: CODE_OF_CONDUCT.md
+[dco]: https://developercertificate.org/
```
Comment (mcollina) [inline CONTRIBUTING.md:91]: why those changes in this file?

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,34 +1,37 @@
...
 <!--- e.g. Chrome 67 -->
-* Browser name and version:
+
+- Browser name and version:
 
 <!--- See http://whatsmyos.com/ -->
 <!--- e.g. OSX 10.11.5 -->
-* Operating system and version:
+
+- Operating system and version:
```
Comment (mcollina) [inline ISSUE_TEMPLATE.md:37]: why those changes in this file?

File: CONTRIBUTING.md (hunk 1 above)
Comment (salmanm) [reply CONTRIBUTING.md:91]: My editor had gone crazy until I fixed `.vscode/settings.json`. No idea why it chose to convert the upper case to lower 😆 
I'm still finalizing this PR, will request reviews once I'm ready.

File: package.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "name": "clinic",
   "description": "Clinic.js diagnoses your performance issues",
   "repository": "nearform/node-clinic",
-  "version": "7.0.0",
+  "version": "8.0.0",
```
Comment (mcollina) [inline package.json:5]: Please revert this, we will bump the version on release.

Comment (salmanm) [reply package.json]: O ok, done.

**Review by jasnell — APPROVED** [review]

**Review by conoro — COMMENTED** [review]
LGTM

File: README.md [hunk 4]
```diff
@@ -103,7 +94,6 @@ For more information use the `--help` option:
 clinic doctor --help
 clinic bubbleprof --help
 clinic flame --help
-clinic upload --help
 clinic ask --help
```
Comment (conoro) [inline README.md:97]: Clinic Ask should be removed here too.

**Pull request merged**